---
title: "Kinase off target prediction"
subtitle: "Articles about kinase off target prediction"
categories:
- Kinome
toc: false
---

## QSAR

* [2021 Collaborative Profile-QSAR: A Natural Platform for Building Collaborative Models among Competing Companies](https://doi.org/10.1021/acs.jcim.0c01342 ){target="_blank"}
* [2017 Profile-QSAR 2.0: Kinase Virtual Screening Accuracy Comparable to Four-Concentration IC50s for Realistically Novel Compounds](https://doi.org/10.1021/acs.jcim.7b00166){target="_blank"}
* [2017 Quantitative Structure–Activity Relationship Modeling of Kinase Selectivity Profiles](https://doi.org/10.3390/molecules22091576 ){target="_blank"}
* [2013 Descriptor-Based Profile Analysis of Kinase Inhibitors to Predict Inhibitory Activity and to Grasp Kinase Selectivity](https://doi.org/10.5012/bkcs.2013.34.9.2680 ){target="_blank"}
* [2011 Kinase-Kernel Models: Accurate In silico Screening of 4 Million Compounds Across the Entire Human Kinome](https://doi.org/10.1021/ci200314j){target="_blank"}
* [2011 Profile-QSAR: A Novel meta-QSAR Method that Combines Activities across the Kinase Family To Accurately Predict Affinity, Selectivity, and Cellular Activity](https://doi.org/10.1021/ci1005004){target="_blank"}
* [2009 QSAR Models for Predicting the Similarity in Binding Profiles for Pairs of Protein Kinases and the Variation of Models between Experimental Data Sets](https://doi.org/10.1021/ci900176y){target="_blank"}

## Others
* [2023 Large-Scale Modeling of Sparse Protein Kinase Activity Data](https://doi.org/10.1021/acs.jcim.3c00132 ){target="_blank"}
* [2022 KiSSim: Predicting Off-Targets from Structural Similarities in the Kinome](https://doi.org/10.1021/acs.jcim.2c00050){target="_blank"}
* [2022 Active Site Sequence Representations of Human Kinases Outperform Full Sequence Representations for Affinity Prediction and Inhibitor Generation: 3D Effects in a 1D Model](https://doi.org/10.1021/acs.jcim.1c00889 ){target="_blank"}
* [2022 On the Choice of Active Site Sequences for Kinase-Ligand Affinity Prediction](https://doi.org/10.1021/acs.jcim.2c00840 ){target="_blank"}
* [2022 Global Analysis of Deep Learning Prediction Using Large-Scale In-House Kinome-Wide Profiling Data](https://doi.org/10.1021/acsomega.2c00664 ){target="_blank"}
* [2022 Differentiating Inhibitors of Closely Related Protein Kinases with Single- or Multi-Target Activity via Explainable Machine Learning and Feature Analysis](https://doi.org/10.3390/biom12040557 ){target="_blank"}
* [2022 Kinome-wide polypharmacology profiling of small molecules by multi-task graph isomorphism network approach](https://doi.org/10.1016/j.apsb.2022.05.004 ){target="_blank"}
* [2022 Improving the Prediction of Potential Kinase Inhibitors with Feature Learning on Multisource Knowledge](https://link.springer.com/article/10.1007/s12539-022-00523-1){target="_blank"}
* [2022 Exploring kinase family inhibitors and their moiety preferences using deep SHapley additive exPlanations](https://doi.org/10.1186/s12859-022-04760-5 ){target="_blank"}
* [2022 Searching Geometric Patterns in Protein Binding Sites and Their Application to Data Mining in Protein Kinase Structures](https://doi.org/10.1021/acs.jmedchem.1c01046){target="_blank"}
* [2022 Structure- and Similarity-Based Survey of Allosteric Kinase Inhibitors, Activators, and Closely Related Compounds](https://doi.org/10.1021/acs.jmedchem.0c02076){target="_blank"}
* [2022 Principles of Kinase Allosteric Inhibition and Pocket Validation](https://doi.org/10.1021/acs.jmedchem.2c00073){target="_blank"}
* [2022 Exploring the kinase-inhibitor fragment interaction space facilitates the discovery of kinase inhibitor overcoming resistance by mutations](https://doi.org/10.1093/bib/bbac203){target="_blank"}
* [2020 Deep Learning Enhancing Kinome-Wide Polypharmacology Profiling: Model Construction and Experiment Validation](https://doi.org/10.1021/acs.jmedchem.9b00855){target="_blank"}
* [2020 Data structures for computational compound promiscuity analysis and exemplary applications to inhibitors of the human kinome](https://doi.org/10.1007/s10822-019-00266-0){target="_blank"}
* [2020 Identifying representative kinases for inhibitor evaluation via systematic analysis of compound-based target relationships](https://doi.org/10.1016/j.ejmech.2020.112641){target="_blank"}
* [2020 Kinase inhibitor data set for systematic analysis of representative kinases across the human kinome](https://doi.org/10.1016/j.dib.2020.106189 ){target="_blank"}
* [2020 IDDkin: network-based influence deep diffusion model for enhancing prediction of kinase inhibitors](https://doi.org/10.1093/bioinformatics/btaa1058){target="_blank"}
* [2020 Revealing Acquired Resistance Mechanisms of Kinase-Targeted Drugs Using an on-the-Fly, Function-Site Interaction Fingerprint Approach](https://doi.org/10.1021/acs.jctc.9b01134 ){target="_blank"}
* [2021 ProfKin: A comprehensive web server for structure-based kinase profiling](https://doi.org/10.1016/j.ejmech.2021.113772){target="_blank"}
* [2021 Using the Structural Kinome to Systematize Kinase Drug Discovery](https://www.intechopen.com/chapters/78606){target="_blank"}
* [2021 Systematic assessment of structure-promiscuity relationships between different types of kinase inhibitors](https://doi.org/10.1016/j.bmc.2021.116226){target="_blank"}
* [2019 Computational analysis of kinase inhibitor selectivity using structural knowledge](https://doi.org/10.1093/bioinformatics/bty582 ){target="_blank"}
* [2019 Chemogenomic Analysis of the Druggable Kinome and Its Application to Repositioning and Lead Identification Studies](https://doi.org/10.1016/j.chembiol.2019.08.007 ){target="_blank"}
* [2019 KinomeX: a web application for predicting kinome-wide polypharmacology effect of small molecules](https://doi.org/10.1093/bioinformatics/btz519){target="_blank"}
* [2018 Coevolutionary Landscape of Kinase Family Proteins: Sequence Probabilities and Functional Motifs](https://doi.org/10.1016/j.bpj.2017.10.028 ){target="_blank"}
* [2018 Determination of a Focused Mini Kinase Panel for Early Identification of Selective Kinase Inhibitors](https://doi.org/10.1021/acs.jcim.8b00222){target="_blank"}
* [2018 Modeling Kinase Inhibition Using Highly Confident Data Sets](https://doi.org/10.1021/acs.jcim.7b00729){target="_blank"}
* [2017 Profiling Prediction of Kinase Inhibitors: Toward the Virtual Assay](https://doi.org/10.1021/acs.jmedchem.6b01611){target="_blank"}
* [2017 Protocols for the Design of Kinase-focused Compound Libraries](https://doi.org/10.1002/minf.201700119){target="_blank"}
* [2017 Kinase-Centric Computational Drug Development](https://doi.org/10.1016/bs.armc.2017.08.001){target="_blank"}
* [2016 Structure-guided selection of specificity determining positions in the human Kinome](https://doi.org/10.1186/s12864-016-2790-3 ){target="_blank"}
* [2016 Predictive Models for Fast and Effective Profiling of Kinase Inhibitors](https://doi.org/10.1021/acs.jcim.5b00646){target="_blank"}
* [2015 Assessing protein kinase target similarity: Comparing sequence, structure, and cheminformatics approaches](https://doi.org/10.1016/j.bbapap.2015.05.004 ){target="_blank"}
* [2013 Combinatorial Clustering of Residue Position Subsets Predicts Inhibitor Affinity across the Human Kinome](https://doi.org/10.1371/journal.pcbi.1003087 ){target="_blank"}
* [2013 De Novo Design of Protein Kinase Inhibitors by in Silico Identification of Hinge Region-Binding Fragments](https://doi.org/10.1021/cb300729y ){target="_blank"}
* [2013 Kinase-Kernel Models: Accurate Chemogenomic Method for the Entire Human Kinome](https://doi.org/10.1002/minf.201300091 ){target="_blank"}
* [2012 Dissecting Kinase Profiling Data to Predict Activity and Understand Cross-Reactivity of Kinase Inhibitors](https://doi.org/10.1021/ci200607f){target="_blank"}
* [2011 Using Multiple Microenvironments to Find Similar Ligand-Binding Sites: Application to Kinase Inhibitor Binding](https://doi.org/10.1371/journal.pcbi.1002326 ){target="_blank"}
* [2010 Computational Modeling of Kinase Inhibitor Selectivity](https://doi.org/10.1021/ml1001097 ){target="_blank"}
* [2010 Cross-Reactivity Virtual Profiling of the Human Kinome by X-ReactKIN: A Chemical Systems Biology Approach](https://doi.org/10.1021/mp1002976 ){target="_blank"}





